examples/multiple/Main.elm

```diff
@@ -0,0 +1,221 @@
+module Multiple.Main exposing (main)
+
+import Html exposing (Html)
+import Mouse exposing (Position)
+
+
+-- Library
+
+import Glue exposing (Glue)
+
+
+-- Submodules
+
+import Multiple.Counter as Counter
+import Multiple.BubblingCounter as BubblingCounter
+import Multiple.Moves as Moves
+
+
+counter : Glue Model Counter.Model Msg Counter.Msg
+counter =
+    Glue.createglue
+        { modelsetter = \subModel model -> { model | counter = subModel }
+        , modelgetter = .counter
+        , init = Counter.init
+        , update = Counter.update
+        , view = Counter.view
+        , subscriptions = \_ -> Sub.none
+        , liftmessage = CounterMsg
+        }
+
+
+counter2 : Glue Model Counter.Model Msg Counter.Msg
+counter2 =
+    Glue.createglue
+        { modelsetter = \subModel model -> { model | counter2 = subModel }
+        , modelgetter = .counter2
+        , init = Counter.init
+        , update = Counter.update
+        , view = Counter.view
+        , subscriptions = \_ -> Sub.none
+        , liftmessage = Counter2Msg
+        }
+
+
+moves : Glue Model Moves.Model Msg Moves.Msg
+moves =
+    Glue.createglue
+        { modelsetter = \subModel model -> { model | moves = subModel }
+        , modelgetter = .moves
+        , init = Moves.init
+        , update = Moves.update
+        , view = Moves.view
+        , subscriptions = Moves.subscriptions
+        , liftmessage = MovesMsg
+        }
+
+
+
+-- bubblingcounter has a non-standard type definition can't use createglue use glue instead
+
+
+bubblingcounter : Glue Model BubblingCounter.Model Msg BubblingCounter.Msg
+bubblingcounter =
```

Comment:
I think it might make sense to abstract this pattern as well. I have it implemented already and will open PR.

prototype implementation:

```elm
bubbling :
    { e
        | init : ( subModel, Cmd msg )
        , liftmessage : subMsg -> msg
        , modelgetter : model -> subModel
        , modelsetter : subModel -> model -> model
        , subscriptions : subModel -> Sub msg
        , update : subMsg -> subModel -> ( subModel, Cmd msg )
        , view : subModel -> Html msg
    }
    -> Glue model subModel msg subMsg
bubbling config =
    glue <|
        { model = config.modelsetter
        , init = config.init
        , update =
            \subMsg model ->
                config.modelgetter model
                    |> config.update subMsg
        , view = \model -> config.view <| config.modelgetter model
        , subscriptions = \model -> config.subscriptions <| config.modelgetter model
        }
```

usage:

```elm
bubblingcounter : Glue Model BubblingCounter.Model Msg BubblingCounter.Msg
bubblingcounter =
    Glue.bubbling
        { modelsetter = \subModel model -> { model | bubblingcounter = subModel }
        , modelgetter = .bubblingcounter
        , init = BubblingCounter.init Even
        , update = BubblingCounter.update Even
        , view = BubblingCounter.view BubblingCounterMsg
        , subscriptions = \_ -> Sub.none
        , liftmessage = BubblingCounterMsg
        }
```